data/plugins/quotes/QuotationsPageSource.py

```diff
@@ -82,11 +82,11 @@ def get_random(self):
 
     def get_for_author(self, author):
         return self.get_for_search_url(
-            iri2uri((u"http://www.quotationspage.com/search.php3?Search=&Author=%s" % author).encode('utf-8')))
+            iri2uri(("http://www.quotationspage.com/search.php3?Search=&Author=%s" % author).encode('utf-8')))
```

Comment:
The handling of Unicode strings is the highest risk when porting from Python 2 to 3. A lot of places like this one will need dedicated testing for how would the app behave if we provide non-ascii symbols (e.g. searching for stuff, non-ascii file paths, folder names, non-ascii symbols in quotes, etc., etc.)